StartMenu.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+import pygame
+import configparser
+import json
+import re
+import os
+
+
+class ConfigHandler(object):
```

Comment:
Das ganze Ding ist IMHO viel zu kompliziert. Es ist doch gar nicht klar ob man diese Komplexität später auch tatsächlich braucht. Stichworte KISS und YAGNI. :-)

Eine Klasse die nur statische ”Methoden” enthält ist keine Klasse sondern nur ein Namensraum für Funktionen.  Dafür sind in Python Module vorgesehen.  Das das Ding so statisch ist, macht zum Beispiel Unit-Tests schwierig bis unmöglich.

Irgendwas handhaben/behandeln oder verwalten (Manager) tut letztendlich so ziemlich _jede_ Klasse, darum kann man den Namenszusatz _Handler_ oder _Manager_ eigentlich auch an _jeden_ Klassennamen dran pappen.  Damit ist dieser Zusatz aber sinnlos weil der keine wirkliche Information enthält.  `Configurations` wäre ausreichend.

Wenn das verschachtelt sein soll, würde ich einfach JSON als Konfigurationsformat verwenden.  Der Code das zu unterstützen ist viel einfacher und man kann beliebig tief hierarchisch Informationen speichern.  Falls Kommentare in der Konfigurationsdatei gewünscht sind, könnte man überlegen statt dessen YAML zu verwenden.